src/Core/EspWindow.hh

```diff
@@ -0,0 +1,75 @@
+#ifndef ESPERT_CORE_ESPERTWINDOW_HH_
+#define ESPERT_CORE_ESPERTWINDOW_HH_
+
+#include "esppch.hh"
+#include "Core/Events/Event.hh"
+
+// libs
+#define GLFW_INCLUDE_VULKAN
+#include "GLFW/glfw3.h"
+#include "volk.h"
```

Comment:
I cannot find the source right now, but I have read that you should include volk before including glfw and should't define GLFW_INCLUDE_VULKAN at all.